episodes/09-branches.md

```diff
@@ -123,13 +123,13 @@ Our current repository looks something like this:
 ```
 
 All our commits, the circles with commit IDs, are on the main branch,
-the horizontal grey line.
+the horizontal blue line.
```

Comment:
```suggestion
the top horizontal line.
```

Reply:
I realised this colour is different in light vs dark mode so referring to it's position instead :)